packages/esm-patient-attachments-app/src/constants.ts

```diff
@@ -0,0 +1 @@
+export const allowedExtensionsGlobalProperty: string = 'attachments.allowedFileExtensions';
```

Comment:
I don't think we need a constants file unless this is used in multiple places. Just keep this local to the hook that uses it.